jmxtrans-core/src/main/java/com/googlecode/jmxtrans/model/Server.java

```diff
@@ -86,7 +86,7 @@ public Server(
 			@JsonProperty("queries") List<Query> queries) {
 		this.alias = resolveProps(alias);
 		this.host = resolveProps(host);
-		this.port = port;
+		this.port = resolveProps(port);
```

Comment:
Could you please add a test case to validate the change ? And to make sure we do not re-introduce this bug again ?